server/routes/images.go

```diff
@@ -107,3 +108,31 @@ func HandleImageUpload(w http.ResponseWriter, r *http.Request) {
 
 	http.Redirect(w, r, imageURL, http.StatusFound)
 }
+
+func HandleImageDelete(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+
+	userService := services.Locator.UserService()
+
+	userID := userService.GetCurrentUserID(ctx)
+
+	if userID == "" {
+		w.WriteHeader(http.StatusUnauthorized)
+		return
+	}
+
+	bucket := config.ImageBucket()
+
+	vars := mux.Vars(r)
+	image := vars["imageID"]
+
+	storageClient := services.Locator.StorageClient()
+
+	err := storageClient.Delete(ctx, bucket, image)
+	if err != nil {
+		serveError(ctx, w, err)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
```

Comment:
Nit: you usually respond with `http.StatusNoContent` and, obviously, no content for DELETE requests

Reply:
I think this also applies to profile pic PUT

Reply:
For the PUT, we'll need to return the new Profile Pic URL, so it returns the updated user

Reply:
Why do we need to return the image URL from the actual PUT request? We should save it internally, yes, but I don't think we need to return it right away

Reply:
The user's profile pic is displayed on their page, so to display the new image, we need the new URL. I guess we could refresh the page after the put, which would trigger the get request to get the info from the back, but getting the results from the put is less effort :P

Reply:
Ok, I see now. Maybe we'll add a GET for the profile pic later, so we don't need to know the exact GCS url